indico/modules/events/registration/models/forms.py

```diff
@@ -528,7 +547,7 @@ def get_registration(self, user=None, uuid=None, email=None):
             return user.registrations.filter_by(registration_form=self).filter(~Registration.is_deleted).first()
         if uuid:
             try:
-                UUID(hex=uuid)
+                uuid.UUID(hex=uuid)
```

Comment:
Ah crap, this change collides with the `uuid` arg of this function. Let's go back to your `SQL_UUID` approach which is better in that case (since renaming the kwarg would be much uglier)

Reply:
Updated.